### DISCUSSION TITLE
GCP problemi dopo il 16 giugno 2024

### DISCUSSION THREAD

**Discussion by texindustries** [indomus/forum#3129] [GENERALE - Discussioni generiche]
Ciao a tutti
qualcuno è riuscito a ha avuto problemi con l'integrazione in Google Home?
dal 16 giugno c'è stato un cambio di policy e ci sono le istruzioni per sistemarlo, ma non essendoci un'organizzazione nella dashboard del progetto no è possibile attuare queste modifiche. 
se qualcuno ha qualche dritta, mi sarebbe d'aiuto... ora come ora home assistant non è vista da google home o dai smart speaker di google... eh si sarebbe una buona occasione per creare smart speaker custom con open ai :)

**Comment by Assistnew** [comment]
Buon pomeriggio,
stesso problema con GCP. Ne approfitto per chiedere dove sono le istruzioni aggiornate per il cambio policy... perchè al momento non funziona nulla. Sarebbe necessario, se possibile, un aggiornamento dell'ottima guida indomus ;-)
Grazie!

**Reply by snakuzzo** [reply]
Quello a cui vi riferite è probabilmente [questo](https://cloud.google.com/blog/products/identity-security/automatically-disabling-leaked-service-account-keys-what-you-need-to-know)

- Andate sulla console GCP.
- Selezionate il project che avete creato
- Andate nella sezione [Service Account](https://console.cloud.google.com/iam-admin/serviceaccounts) (o Account di Servizio in italiano)
- Cliccate sui dettagli del service account creato per l'integrazione e poi sul tab Keys (o Chiavi in italiano)
- Se la chiave non risultasse abilitata, vuol dire che è stata disabilitata per questioni di sicurezza

![image](https://github.com/user-attachments/assets/849b81b0-0612-4fcc-a8e9-0420cd2ac168)

**Reply by snakuzzo** [reply]
Per chiarire...
non c'è alcuna guida da aggiornare o soluzioni da fornire. Questa è un'**azione preventiva** che ha attuato **giustamente** Google per la sicurezza dei propri clienti. Ora sta a voi scegliere cosa fare ed eventualmente seguire o meno le indicazioni fornite da Google nell'articolo linkato.

Io però, al posto vostro, due domande sul perché quella chiave sia finita su qualche repository pubblico me la farei.

**Comment by Assistnew** [comment]
Il service account risulta abilitato. Ho provato anche a reimpostare tutto da zero, ma il problema permane.
Non capisco proprio dov'è l'errore, anche perchè ha sempre funzionato correttamente fino al cambio policy.
Grazie comunque

**Reply by snakuzzo** [reply]
correggo sopra...devi controllare la chiave, non il service account...

**Comment by Assistnew** [comment]
Attiva anche la chiave.
![key](https://github.com/user-attachments/assets/b5f29c18-1c0c-4945-9fbd-66a930074c4d)

**Reply by snakuzzo** [reply]
se sicuro di aver selezionato il service account giusto ?
Se sì ed  è tutto attivo, il problema non è legato a questa cosa

**Reply by snakuzzo** [reply]
e per favore, fai reply sotto l'ultimo messaggio...non come hai fatto prima e adesso creando nuovi thread sotto la stessa discussione.
Grazie.

**Reply by texindustries** [reply]
per il mio caso, la cosa si è risolta da sola… non so se era Google un po' fuori fase, però adesso mi funziona senza nessuna particolare azione. Devo solo passare a FTTH perché con FTTC ho spesso dei ritardi in upload e Google home non ha un timeout abbastanza lungo e può capitare che non mi esegui dei comandi vocali

**Reply by Assistnew** [reply]
si tutto attivo. Ok, grazie comunque per la disponibilità.
